### PR TITLE
AWS Load Balancer controller and bitnami PSQL hel chart 

### PR DESCRIPTION
**Pull request by SBraunovicOmega**

### CODE REVIEW

File: infrastructure/helm.tf [hunk 1]
```diff
@@ -0,0 +1,79 @@
+resource "helm_release" "load_balancer_controller" {
+  name             = "stefan-loadbalancer-controller"
+  repository       = "https://aws.github.io/eks-charts"
+  chart            = "aws-load-balancer-controller"
+  namespace        = "vegait-training"
```
Comment (Igor-Knezevic-VegaIT) [inline infrastructure/helm.tf]: lb controller should be in it's own namespace

Comment (SBraunovicOmega) [reply infrastructure/helm.tf]: So, does the namespace need to be what I put in the irsa load balance module?

File: infrastructure/helm.tf [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    value = module.vpc.vpc_id
+  }
+  set {
+    name  = "defaultTargetType"
+    value = "ip"
+  }
+}
+
+
+resource "kubernetes_persistent_volume_claim" "postgres_pvc" {
```
Comment (Igor-Knezevic-VegaIT) [inline infrastructure/helm.tf]: You don't define this in terraform as a separate resource, you should define storage class which will helm chart utilise to privison pvc inside kubernetes.

File: infrastructure/helm.tf [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+resource "helm_release" "bitnami_psql" {
+  name       = "stefan-binami-psgl"
+  repository = "oci://registry-1.docker.io/bitnamicharts/postgresql"
+  chart      = "my-release"
+  namespace  = "vegait-training"
+
+  set {
+    name  = "auth.username"
+    value = "postgres"
```
Comment (Igor-Knezevic-VegaIT) [inline infrastructure/helm.tf]: values should come from secrets manager

File: infrastructure/helm.tf [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+  set {
+    name  = "auth.database"
+    value = "to-do"
+  }
+  set {
+    name  = "containerPorts.postgresql"
+    value = 5432
+  }
+  set {
+    name  = "persistence.existingClaim"
```
Comment (Igor-Knezevic-VegaIT) [inline infrastructure/helm.tf]: you don't want this, you want helm chart to provision storage via pvc and storage class.

**Review by Igor-Knezevic-VegaIT — APPROVED** [review]

**Pull request merged**